pom.xml

```diff
@@ -227,7 +227,25 @@
                 <goal>sort</goal>
               </goals>
             </execution>
-          </executions>
+          </executions>                                             
+        </plugin>
+        <plugin>
+          <groupId>au.com.acegi</groupId>
+          <artifactId>xml-format-maven-plugin</artifactId>
+          <version>3.1.2</version>
+          <executions>                                                                                                                                                                               
+           <execution>
+             <id>xml-format</id>
+             <goals>
+           <goal>xml-format</goal>                         
+          </goals>
+         </execution>
+        </executions>
+       <configuration>
```

Comment:
Hmm, how is this new plugin executed on the XML files? It seems to me that you have only added this to `<pluginManagement>`, right? Shouldn't you add it to some `<plugins>`

Reply:
@bashir2 yes sure thanks,
 i have added it to the `<plugins>`